OpenRA.sln.DotSettings

```diff
@@ -1,4 +0,0 @@
-<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
-	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/DEFAULT_PRIVATE_MODIFIER/@EntryValue">Implicit</s:String>
```

Comment:
Is it worth keeping this config line since RS doesn't seem to support the `omit_if_default` from the editorconfig? We could still remove the other lines.

Reply:
ReSharper doesn't seem to respect that one either, which was the main reason I started down this path - it has been (and still continues to, even on this branch) suggesting I explicitly write the access modifiers. At least now if you fall into its trap you get a warning that you should remove them 🤷‍♂️